text/0005-send-report.md

```diff
@@ -0,0 +1,67 @@
+- Feature Name: send_report_api
+- Start Date: 2021-08-02
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
+
+# Summary
+[summary]: #summary
+
+Report API will receive reports from users about tokens, transactions, etc.
+
+# Motivation
+[motivation]: #motivation
+
+We are concerned about scammers, abuses, etc. As we have a metadata API that will carry information about tokens and
+transactions, we need a way where users can report tokens used in scams and respective transactions.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Report will be received as a POST and sent to a special email from hathor.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+**Endpoint:**
```

Comment:
It seems this is the Guide Level explanation. For the low level, I think we should explain what the API will do. 

I feel we should internally save an S3 file for each report, add a report counter to the token/transaction, and send an email. Maybe the email will be disabled in the future in case we start to receive too many reports.